inst/dev-guide/dg_tabulation.Rmd

```diff
@@ -0,0 +1,176 @@
+---
+title: "Tabulation"
+author: "Davide Garolini"
+date: '`r Sys.Date()`'
+output:
+  html_document:
+    theme: spacelab
+    toc: true
+    toc_float:
+      collapsed: false
+editor_options: 
+  chunk_output_type: console
+knit: (function(inputFile, encoding) {
+  rmarkdown::render(inputFile, encoding = encoding, output_dir = ".")})
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Disclaimer
+
+Any code or prose which appears in a version of this vignette on the `main` branch of the repository may reflect a specific state of things that can be more or less recent. This guide reflects very important pieces of the split machinery that are unlikely to change. We anyway invite the reader to think always that the current code may have drifted from the following document, and it is always the best practice to read directly the code on `main`. (xxx we should insert it automatically)
```

Comment:
I think we should insert this automatically. @edelarua @ayogasekaram you remember how best to add this from a common source?

Reply:
I think the way we did this previously (in `tern`?) only worked for Roxygen documentation, i.e. creating a doc with the blurb and reusing it with a few words changed in multiple places.